cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -305,6 +299,9 @@ pub fn define(
         .is_branch(true),
     );
 
+    // These are the instructions which br_table legalizes to: they operate on real pointers, so
+    // their type variables are more constrained.
```

Comment:
The word "real" here is a little confusing. And technically the index isn't a pointer; it's just a pointer-sized integer. How about saying something like "they perform address computations, using pointer-sized integers, so their type variables are more constrained"?